src/main/java/com/axeiya/gwtckeditor/client/CKConfig.java

```diff
@@ -646,10 +647,11 @@ private native void setNativeEntitiesLatin(boolean entitiesLatin) /*-{
 
 	private native void setToolbarObject(JavaScriptObject toolbarArray) /*-{
 //		this.@com.axeiya.gwtckeditor.client.CKConfig::config.toolbar_temp = toolbarArray;
-		this.@com.axeiya.gwtckeditor.client.CKConfig::config.toolbar = [
-    ['Undo', 'Redo'],
-    ['EmojiPanel']
-];
+		this.@com.axeiya.gwtckeditor.client.CKConfig::config.toolbar = toolbarArray;
+//		[
+//    ['Undo', 'Redo'],
+//    ['EmojiPanel']
+//];
 	}-*/;
```

Comment:
@raviteja-kaarya make sure to test text and email customer Pop ups with Emoji DSO.

Reply:
@mohittyagi6677 This testing is done. Merging PR